featurehub_config.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import annotations
+
+from client_context import ClientContext, ClientEvalFeatureContext, ServerEvalFeatureContext
+from edge_service import EdgeService
+from featurehub_repository import FeatureHubRepository
+
+
+class FeatureHubConfig:
+    # readyness: Readyness;
+
+    _api_keys = []
+    _api_key: str
+    _edge_url: str
+    _client_eval: bool
+
+    def __init__(self, edge_url, api_key):
+        self._repository = FeatureHubRepository
+        self._edge_url = edge_url
+        self._api_key = api_key
+
+        if not edge_url or not api_key:
+            raise TypeError('apiKey and edge_url must not be null')
+
+        self._api_keys = [api_key]
+
+        self._client_eval = '*' in self._api_key
+
+        if not self._edge_url[-1] == '/':
+            self._edge_url += '/'  # think this will fail as strings are immutable
+
+        self._url = self._edge_url + 'features/' + self._api_key
```

Comment:
you probably want to take the opportunity to change this and not store it, its a historical API for typescript. This URL is only correct for SSE, but it was always there in TS, so I would recommend you remove it.